packages/relay-compiler/language/javascript/FindGraphQLTags.js

```diff
@@ -141,46 +114,18 @@ function find(
     },
     TaggedTemplateExpression: node => {
       if (isGraphQLTag(node.tag)) {
-        const template = getGraphQLText(node.quasi);
-        if (validateNames) {
```

Comment:
so validation now occurs at the level of the unparametrized RelayFindGraphQLTags, right?

Reply:
Correct.